.stylelintrc.json

```diff
@@ -1,8 +1,9 @@
 {
   "extends": "stylelint-config-sass-guidelines",
   "plugins": [
-     "stylelint-suitcss",
-     "stylelint-scss"
+    "stylelint-order",
+    "stylelint-suitcss",
+    "stylelint-scss"
```

Comment:
Why the additional plugins?

Reply:
When updating stylelint to 9.x, they suggest to include [stylelint-order](https://github.com/hudochenkov/stylelint-order/issues/7#issuecomment-278487061)